Templates/SasToken-CSharp/function.json

```diff
@@ -0,0 +1,16 @@
+{
+    "disabled": false,
+    "bindings": [
+        {
+            "authLevel": "function",
+            "name": "req",
+            "type": "httpTrigger",
+            "direction": "in"
```

Comment:
It just restricts the verbs you support. If requests come in for unsupported verbs, a 405 "Method Not Allowed" will be returned